e2e/container/attach_test.go

```diff
@@ -37,7 +37,6 @@ func TestAttachInterrupt(t *testing.T) {
 	// todo(laurazard): make this test work w/ dind over ssh
 	skip.If(t, strings.Contains(os.Getenv("DOCKER_HOST"), "ssh://"))
 
-	// if
```

Comment:
😂 🤔 🤷

Reply:
oh ☝️ It's ok (or "better') to keep this fix as a  separate commit.  It's one of those that may come back and bit you in the back if you try to apply changes to different branches (so having it separate can allow "skip it")

Reply:
For sure! This one was going to stay separate.